### PR TITLE
Keep closed jobs posted

### PR DESCRIPTION
**Pull request by lsarni**
## What changes were made
- Added field `closed` to jobs
- Show banner on top of closed jobs
- Hide "Apply" button if the job is closed
- Replace `callToActionTitle` field with translation

## How to test or check results

https://deploy-preview-710--voorhoede-website.netlify.app/en/jobs/senior-front-end-developer/?preview=true&previewSecret=blue-unsalted-ravioli

image

## Checks
- [x] All content model changes are done through [scripted migrations](../readme.md#scripted-migrations)
- [x] Appointed PR reviewers

### CODE REVIEW

File: src/pages/[language]/jobs/[slug].vue [hunk 1]
```diff
@@ -26,19 +41,20 @@
         </template>
         <footer class="page-job__footer button-group">
           <h2 class="page-job__title h3">
-            {{ data.page.callToActionTitle }}
```
Comment (lsarni) [inline src/pages/[language]/jobs/[slug].vue:29]: It was always the same text, Vera didn't mind moving it to a translation

File: migrations/1688126055_closedJobs.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  await client.fields.update(newFields["10413779"], { position: 0 });
+
+  console.log("Create new translations");
+
+  await client.items.create({
+    item_type: { type: 'item_type', id: '55443' },
+    key: 'apply_closed',
+    value: {
+      en: 'This vacancy is currently closed. Please check back later or apply to one of our other vacancies.',
+      nl: 'Deze vacature is momenteel gesloten. Kom later nog eens terug of solliciteer op een van onze andere vacatures.',
```
Comment (lsarni) [inline migrations/1688126055_closedJobs.ts:40]: Can you propose Dutch translations?

Comment (Frankwarnaar) [reply migrations/1688126055_closedJobs.ts:40]: Dutch translations are looking good already! :)

File: migrations/1688126055_closedJobs.ts [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  console.log("Creating new fields/fieldsets");
+
+  console.log(
+    'Create Boolean field "Is closed?" (`closed`) in model "Job" (`job`)'
+  );
+  newFields["10413779"] = await client.fields.create("38231", {
+    label: "Is closed?",
+    field_type: "boolean",
+    api_key: "closed",
+    hint: "Closed jobs will remain online for SEO optimization reasons but will not be directly listed on the website. A banner explaining the position is closed will appear on the top of the page.",
```
Comment (Siilwyn) [inline migrations/1688126055_closedJobs.ts:15]:
```suggestion
    hint: "Closed jobs will remain online for search engine optimization reasons but will not be directly listed on the website. A banner explaining the position is closed will appear on the top of the page.",
```
Nitpick

File: migrations/1688126055_closedJobs.ts [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+  console.log("Creating new fields/fieldsets");
+
+  console.log(
+    'Create Boolean field "Is closed?" (`closed`) in model "Job" (`job`)'
+  );
+  newFields["10413779"] = await client.fields.create("38231", {
+    label: "Is closed?",
+    field_type: "boolean",
+    api_key: "closed",
```
Comment (Siilwyn) [inline migrations/1688126055_closedJobs.ts:14]: Thoughts on naming this `is_closed` think it's common naming pattern for booleans?

**Review by Siilwyn — CHANGES_REQUESTED** [review]
Should we hide closed jobs from the jobs overview page too? Feels a bit weird to list them to then discover it's closed.

File: migrations/1688126055_closedJobs.ts (hunk 3 above)
Comment (lsarni) [reply migrations/1688126055_closedJobs.ts:15]: It was giving "chai tea"

Comment (Siilwyn) [reply migrations/1688126055_closedJobs.ts:15]: https://media.tenor.com/Pvb52tjijZMAAAAC/spider-man-miles-morales.gif

**Review by Frankwarnaar — APPROVED** [review]

**Pull request merged**